source/adapters/level_zero/CMakeLists.txt

```diff
@@ -22,8 +22,20 @@ endif()
 if (NOT DEFINED LEVEL_ZERO_LIBRARY OR NOT DEFINED LEVEL_ZERO_INCLUDE_DIR)
     message(STATUS "Download Level Zero loader and headers from github.com")
 
+    # Workaround warnings/errors for Level Zero build
+    set(CMAKE_CXX_FLAGS_BAK "${CMAKE_CXX_FLAGS}")
+    if (UNIX)
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-but-set-variable")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pedantic")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-stringop-truncation")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-parameter")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-c++98-compat-extra-semi")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unknown-warning-option")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-non-virtual-dtor")
```

Comment:
As of https://github.com/intel-innersource/libraries.compute.oneapi.level-zero.loader/commit/72de09cf8d99f683f5eaf06590f6d037becd5fb8 you should not need at least non-virtual-dtor

Reply:
thanks @nrspruit . Seems like it is not needed, based on my local testing. Removed it.